receptorctl/receptorctl/cli.py

```diff
@@ -97,10 +97,16 @@ def status(ctx):
     ads = status.pop('Advertisements', None)
     if ads:
         print()
-        print(f"{'Node':<{longest_node}} Service   Last Seen            Tags")
+        print(f"{'Node':<{longest_node}} Service   Type        Last Seen            Tags")
         for ad in ads:
             time = dateutil.parser.parse(ad['Time'])
-            print(f"{ad['NodeID']:<{longest_node}} {ad['Service']:<8}  {time:%Y-%m-%d %H:%M:%S}  ", end="")
+            if ad['ConnType'] == 0:
+                conn_type = 'PacketConn'
+            elif ad['ConnType'] == 1:
+                conn_type = 'ConnNonTLS'
+            elif ad['ConnType'] == 2:
+                conn_type = 'ConnTLS'
```

Comment:
Conn and PacketConn are only meaningful to a Go developer.  For end-user-visible status, we should call these something like Datagram, Stream and TLS_Stream (or StreamTLS?).  And maybe the constants in the Go code should reflect these names, just for consistency.

Reply:
renamed them to `Datagram`, `Stream`, and `StreamTLS`